### ISSUE TITLE
NPE upon closing all open files

### ISSUE THREAD

**Issue by BreadMoirai** [TsundereBug/discord-intellij#1]
```java
null
java.lang.NullPointerException
	at com.tsunderebug.discordintellij.FileChange.selectionChanged(FileChange.scala:19)
	at sun.reflect.GeneratedMethodAccessor56.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:117)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:443)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:404)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:393)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:381)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:374)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:45)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:231)
	at com.sun.proxy.$Proxy109.selectionChanged(Unknown Source)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.a(FileEditorManagerImpl.java:1548)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$5$1.run(FileEditorManagerImpl.java:1031)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.a(FocusManagerImpl.java:640)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.o(FocusManagerImpl.java:624)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.c(FocusManagerImpl.java:594)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.access$200(FocusManagerImpl.java:61)
	at com.intellij.openapi.wm.impl.FocusManagerImpl$IdleRunnable.runEdt(FocusManagerImpl.java:107)
	at com.intellij.openapi.util.EdtRunnable.lambda$run$0(EdtRunnable.java:27)
	at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2760)
	at com.intellij.openapi.util.EdtRunnable.run(EdtRunnable.java:25)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:237)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
``

**Comment by s5bug** [comment]
In theory fix:

When new file is `null`, say the user is not editing anything.

**Comment by s5bug** [comment]
Fixed in update 1.2.2 which should arrive on the IntelliJ repo in 1-2 days.
